package/common_init/overlay/etc/init.d/S80endpoint_router_sbp

```diff
@@ -17,6 +17,7 @@ if [[ "$ins_output_mode" == "Loosely Coupled" ]]; then
   router_config="/etc/endpoint_router/sbp_router_smoothpose.yml"
```

Comment:
Good start.

Above we'll need to change from `starling_daemon_enabled=$(query_config --section system --key enable_starling_daemon)` to `starling_daemon_enabled=$(query_config --section experimental_flags --key starling_on_linux)`.

Then in piksi_system_daemon we need to remove the settings code for `system.enable_starling_daemon`.

Reply:
for reference on the PFWP side of things: https://github.com/swift-nav/piksi_firmware_private/pull/2389